src/Packetery/Module/Order/PacketCanceller.php

```diff
@@ -191,6 +192,7 @@ public function cancelPacket( Entity\Order $order ): void {
 			];
 
 			$this->logger->add( $record );
+			$order->updateApiErrorMessage( $result->getFaultString() );
```

Comment:
Pokud tam není faultString, vrací getFaultString null, updateApiErrorMessage stačí tedy volat jednou pro obě větve. Zkontroluj prosím všechna použití updateApiErrorMessage.